src/io/perun.clj

```diff
@@ -129,26 +177,17 @@
   "Parse markdown files
 
    This task will look for files ending with `md` or `markdown`
-   and add a `:content` key to their metadata containing the
+   and add a `:parsed` key to their metadata containing the
    HTML resulting from processing markdown file's content"
```

Comment:
It seems that this also writes out HTML files to the location related, is that correct?